config/plugins.js

```diff
@@ -53,6 +53,13 @@ module.exports = root => [
             width: 980,
           },
         },
+        {
+          resolve: "gatsby-remark-images",
+          options: {
+            maxWidth: 900,
+            linkImagesToOriginal: false,
```

Comment:
not sure about these two options.

I went with 900px since I saw that our posts have a max width of 850 or so.
the linking part just felt "right", but I'm open to feedback. By default, it creates an hover effect and a link to open the image on a separate tab

Reply:
The max width in the design is set to 834px, we can bind to that value (I'll leave the suggestion).

As for the linking part, I agree with you. That is not an expected behavior in our blog from anyone in the team, and has never been mentioned as a need. I don't see any reason to include it. :+1: